src/ExtendedXmlSerializer/IExtendedXmlSerializerConfig.cs

```diff
@@ -27,12 +27,14 @@ namespace ExtendedXmlSerialization
 {
     public interface IExtendedXmlSerializerConfig
     {
-        Type Type { get; }
-        int Version { get; set; }
+        Type Type { get; } // Consider making obsolete as logic is now contained within implementation.
+        int Version { get; set; } // Consider making getter only, defined by implementation.
```

Comment:
I haven't idea what for someone want use this property. In standard implementation you don't need use it. Am I wrong ?
Version property is old. Before I've published ExtendedXmlSerializer I've had not configuration ;) This is residue.